tests/Kernel/fixtures/rendering.yml

```diff
@@ -1371,3 +1371,36 @@
       '.ecl-timeline2__item:nth-child(4) .ecl-timeline2__content': "Lorem Ipsum 4"
       '.ecl-timeline2__item:nth-child(5) .ecl-timeline2__content': "Lorem Ipsum 5"
       '.ecl-timeline2__item:nth-child(6) .ecl-timeline2__content': "Lorem Ipsum 6"
+- array:
+    '#type': pattern
+    '#id': icons_with_text
+    '#fields':
+      items:
+        -
+          icon: "file"
+          text: "Culture | Innovation"
```

Comment:
What will happen if don't provide one of these?

Reply:
Updated behavior. If an icon or text is empty, we remove this list item.
@ademarco What do you think, is this correct behavior?

Reply:
I would not display the item only if the text is empty, while if the icon is empty we can default to the `document` one, so we always display one.

Reply:
I still don't see the fallback to `document` icon if none is specify, did we do this?

Reply:
Done. But we don't have, unfortunately, `document` icon. So I used `file` icon.